Sources/_StringProcessing/Engine/MEBuiltins.swift

```diff
@@ -1,13 +1,221 @@
-
+@_implementationOnly import _RegexParser // For AssertionKind
 
 extension Processor {
+  mutating func matchBuiltin(
+    _ cc: _CharacterClassModel.Representation,
+    _ isInverted: Bool,
+    _ isStrictAscii: Bool
+  ) -> Bool {
+    guard let c = load() else {
+      signalFailure()
+      return false
+    }
+
+    var matched: Bool
+    var next = input.index(after: currentPosition)
+    switch cc {
+    case .any, .anyGrapheme: matched = true
```

Comment:
Can you test this against `"\n"`? I don't recall if the `any` here is `dot` or really any character.

Reply:
Are these affected by strict ASCII? If so, can we hoist that check? If not, can we make a local

```
var asciiCheck: Bool { c.isASCII || !isStrictAscii }
```

to clean up the switch cases?